docs/api/portable-stories/portable-stories-vitest.mdx

```diff
@@ -161,16 +161,54 @@ draft: true
   ## setProjectAnnotations
 
   This API should be called once, before the tests run, typically in a [setup file](https://vitest.dev/config/#setupfiles). This will make sure that whenever `composeStories` or `composeStory` are called, the project annotations are taken into account as well.
+  
+  These are the configurations needed in the setup file:
+  - preview annotations: those defined in `.storybook/preview.ts`
+  - addon annotations (optional): those exported by addons
+  - beforeAll: code that runs before all tests
+  - testingLibraryRender: the render function from Testing Library
 
-  ```ts
-  // setup-portable-stories.ts
-  // Replace <your-renderer> with your renderer, e.g. react, vue3, svelte
-  import { setProjectAnnotations } from '@storybook/<your-renderer>';
-  import * as addonAnnotations from 'my-addon/preview';
-  import * as previewAnnotations from './.storybook/preview';
+  {/* prettier-ignore-start */}
 
-  setProjectAnnotations([previewAnnotations, addonAnnotations]);
-  ```
+  <CodeSnippets path="portable-stories-vitest-set-project-annotations.md" />
+
+  {/* prettier-ignore-end */}
+
+  <Callout variant="warning">
```

Comment:
@yannbf — I'm not sure if this callout is more or less appropriate/helpful than the one in the `3. Play` section below... but we need to consolidate (and maybe remove the second one?).